elliptic-curve/Cargo.toml

```diff
@@ -29,7 +29,7 @@ zeroize = { version = "1", default-features = false }
 
 # optional dependencies
 base64ct = { version = "1", optional = true, default-features = false }
-digest = { version = "0.9", optional = true, default-features = false }
+digest = { version = "0.10", optional = true, default-features = false, features = ["core-api"] }
```

Comment:
`core-api` is enabled by default and it's the only default feature, so you can simply use `digest = "0.10"`.